src/client/components/Checker.tsx

```diff
@@ -29,10 +29,7 @@ export const Checker: FC<CheckerProps> = ({ config }) => {
   }
 
   const onSubmit = (inputVariables: Record<string, string | number>) => {
-    const computedVariables = operations.reduce(
-      evaluateOperation,
-      inputVariables
-    )
+    const computedVariables = operations.reduce(variableReducer, inputVariables)
```

Comment:
A bit conflicted here - function names should be verbs but given you're mostly using this as a lambda here, a noun for a name is somewhat suitable. Leaving this to you.

Reply:
Referring to MDN, where they call it a `reducer`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

As much as I dislike it as well, it might be clearer to follow their convention